test/FunctionalTestUtils/Tests/RequestTelemetryTestsBase.cs

```diff
@@ -31,5 +33,25 @@ public void ValidateBasicRequest(string requestPath, RequestTelemetry expected)
             Assert.True(actual.Duration < (DateTimeOffset.Now - testStart), "duration");
             Assert.Equal(expected.HttpMethod, actual.HttpMethod);
         }
+
+        public void ValidateBasicException(string requestPath, ExceptionTelemetry expected)
+        {
+            DateTimeOffset testStart = DateTimeOffset.Now;
+            var task = this.HttpClient.GetAsync(this.Server.BaseHost + requestPath);
+            task.Wait(TestTimeoutMs);
+            var result = task.Result;
+
+            var items = this.Buffer.Where((item) => { return item is ExceptionTelemetry; });
+            Assert.Equal(1, items.Count());
+            ITelemetry telemetry = items.First();
+            Assert.IsAssignableFrom(typeof(ExceptionTelemetry), telemetry);
+            var actual = (ExceptionTelemetry)telemetry;
+
```

Comment:
Sorry, my [OCD](http://en.wikipedia.org/wiki/Obsessive%E2%80%93compulsive_disorder) compels me to say this :blush:  With Linq and xUnit, this code block could be shortened to 
```c#
ITelemetry telemetry = this.Buffer.OfType<ExceptionTelemetry>().Single();
var actual = Assert.IsType<ExceptionTelemetry>(telemetry);
```